### ISSUE TITLE
Keyboard accessibility

### ISSUE THREAD

**Issue by wonbug** [algolia/hn-search#232]
Would it be possible to make the website function with keyboard only? Right now, the search drop-down menus (stories, popularity, date) functionality cannot be used without a mouse. Thanks!
